Floofbot/Modules/Administration.cs

```diff
@@ -474,15 +482,19 @@ public async Task warnlog([Summary("user")] string user)
                 builder.AddField(":warning: | Formal Warnings:", "\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_\\_");
                 foreach (Warning warning in formalWarnings)
                 {
+                    var HyperLink = "";
```

Comment:
You probably wanted `hyperLink` instead of `HyperLink`? Just to make sure variable names look like names instead of types

Reply:
Fixed